### DISCUSSION TITLE
The prop value with an expression type of JSXEmptyExpression could not be resolved. Please file issue to get this fixed immediately.

### DISCUSSION THREAD

**Discussion by ffMathy** [eslint/eslint#13774] [Q&A]
The prop value with an expression type of JSXEmptyExpression could not be resolved. Please file issue to get this fixed immediately.

**Comment by mdjermanovic** [comment]
(moved to [Discussions](https://github.com/eslint/eslint/discussions))

That [message](https://github.com/jsx-eslint/jsx-ast-utils/blob/4576cde302cd8b9ef40b6dc026b20cb005f2dbea/src/values/expressions/index.js#L61) is coming from `jsx-ast-utils` library, and an issue should be filed in this repository:

https://github.com/jsx-eslint/jsx-ast-utils
